tests/unit/test_business_logic.py

```diff
@@ -112,25 +112,48 @@ def test_workspace_manifest_for_file_not_found(bll):
         workspace.get_manifest_for_file(UrlPath("foo/bar.txt"))
 
 
-def test_get_size():
+def test_get_file_metadata():
     workspace = factories.create_workspace("workspace")
-    assert workspace.get_size(UrlPath("metadata/foo.log")) == 0
+    empty = workspace.get_file_metadata(UrlPath("metadata/foo.log"))
+    assert empty.size is None
+    assert empty.size_mb == ""
+    assert empty.timestamp is None
+    assert empty.modified_at is None
+    assert empty.content_hash is None
```

Comment:
What's the scenario that this is testing?

Reply:
There is no manifest and no file on disk to supply the metadata

Reply:
yep